kuma/javascript/src/l10n.js

```diff
@@ -136,3 +137,32 @@ export function interpolate(s: string, args: Array<any> | { [string]: any }) {
         );
     }
 }
+
+const ELEMENT_REGEXP = /(<\w+\s*\/>)/g;
+/**
+ * Similar to interpolate() it interpolates the given id-string with the given
+ * properties, but it can also work with React Elements. It will replace strings
+ * of the form `<name/>` and return an Array of React Elements.
+ */
+export function Interpolated({
+    id,
+    ...args
+}: {
+    id: string,
+    [key: string]: React.Node
+}): React.Node[] {
+    return id.split(ELEMENT_REGEXP).map((str, i) => {
+        if (!ELEMENT_REGEXP.test(str)) {
```

Comment:
It took some reasoning for me to completely understand what is going on here and in the next conditional :) Maybe it is just me but, perhaps it is worth adding some comments to explain what it does. Something like:

```
/* Tests whether there is an element in the string, for example
   ELEMENT_REGEXP.test("This is just a string"); // false 
   ELEMENT_REGEXP.test("<elem/>"); // true */
```